db/migrate/20210831214137_index_tag_nominations_and_tag_set_nominations.rb

```diff
@@ -0,0 +1,55 @@
+class IndexTagNominationsAndTagSetNominations < ActiveRecord::Migration[5.2]
+  def up
+    if Rails.env.staging? || Rails.env.production?
+      database = TagNomination.connection.current_database
+
+      puts <<~PTOSC
+        Schema Change Command:
+
+        pt-online-schema-change D=#{database},t=tag_nominations \\
+          --alter "ADD INDEX index_tag_nominations_on_tag_set_nomination_id_and_type (tag_set_nomination_id, type)" \\
+          --no-drop-old-table \\
+          -uroot --ask-pass --chunk-size=5k --max-flow-ctl 0 --pause-file /tmp/pauseme \\
+          --max-load Threads_running=15 --critical-load Threads_running=100 \\
+          --set-vars innodb_lock_wait_timeout=2 --alter-foreign-keys-method=auto \\
+          --execute
+
+        Table Deletion Command:
+
+        DROP TABLE IF EXISTS `#{database}`.`_tag_nominations_old`;
+      PTOSC
+    else
+      add_index :tag_nominations, [:tag_set_nomination_id, :type]
```

Comment:
Actually, I added `type` to the index because I think the existing `type, tag_set_nomination_id` index should have been reversed (`tag_set_nomination_id, type`). But I don't want to remove the existing index in this issue just in case I've miscalculated and removing it would slow things down.

(I'm not sure that having the `type` in the index would make that query faster, because both the `unreviewed` and the `parent_tagname` constraint aren't covered by the index. And it'd still have to do a secondary sort by `tagname`, which also isn't covered by the index.)